README.md

```diff
@@ -46,8 +46,12 @@ This project help you in easy way to pull up information about users in your dat
 
 *If You want to run tests:*
 
+* You must be in your scripts folder,
+
 * **python -m unittest -v main_test.TestDigitAddition**, Run a sample test
```

Comment:
You don't need separate lines to run different test.
Find the way to run all the tests at once.